gcm_filters/filter.py

```diff
@@ -280,6 +286,19 @@ class Filter:
 
     def __post_init__(self):
 
+        self.Laplacian = ALL_KERNELS[self.grid_type]
+
+        # Determine whether this is simple fixed factor filter; in that case we need dx_min = 1
+        if issubclass(self.Laplacian, BaseScalarLaplacianWithArea):
+            if self.dx_min != 1:
+                warnings.warn(
+                    "Provided Laplacian is for simple fixed factor filtering, "
+                    "where area-weighted field is filtered on a regular grid with dx = dy = 1 "
+                    "--> dx_min is set to 1",
+                    stacklevel=2,
+                )
+                self.dx_min = 1
```

Comment:
What scenario are you imagining here with this check / warming? Rather than overwriting `dx_min`, why don't we just raise a `ValueError` here and make the user fix it explicitly?